src/lib.rs

```diff
@@ -47,16 +47,16 @@
 //! let mut reader = io::BufReader::new(certfile);
 //!
 //! // Load and return certificate.
-//! let certs = rustls_pemfile::certs(&mut reader).unwrap();
-//! let certs = certs.into_iter().map(rustls::Certificate).collect();
+//! let certs = rustls_pemfile::certs(&mut reader).flat_map(|x| x);
+//! let certs = certs.map(Into::into).collect();
 //!
 //! // Load private key. (see `examples/server.rs`)
 //! let keyfile = File::open("examples/sample.rsa").unwrap();
 //! let mut reader = io::BufReader::new(keyfile);
 //!
 //! // Load and return a single private key.
-//! let keys = rustls_pemfile::rsa_private_keys(&mut reader).unwrap();
-//! let key = rustls::PrivateKey(keys[0].clone());
+//! let keys: Vec<pki_types::PrivatePkcs1KeyDer<'static>> = rustls_pemfile::rsa_private_keys(&mut reader).flat_map(|x| x).collect();
```

Comment:
Use `private_key()` here.

Reply:
I searched though the source code but I doesn't know how to use that, well I'd love to see if there's a concrete example how.

Reply:
Hmm, here's the type signature:
```
pub fn private_key(rd: &mut dyn io::BufRead) -> Result<Option<PrivateKeyDer<'static>>, io::Error> 
```
Look like we may need to have a bit of a code change here to account for that Option. Maybe a `filter_map(Result::ok)` would suffice...?

Reply:
Done